panther_driver/src/driver_node.py

```diff
@@ -258,6 +258,8 @@ def driverNode():
             try:
                 Idriv1 = float(front_controller.sdo["Qry_BATAMPS"][1].raw)/10
                 Idriv2 = float(rear_controller.sdo["Qry_BATAMPS"][1].raw)/10
+                V_driv1 = float(front_controller.sdo[0x210D][2].raw)/10
+                V_driv2 = float(rear_controller.sdo[0x210D][2].raw)/10
```

Comment:
How about spaces after division?

Reply:
:+1:

Reply:
Do we need this try-except block since we don't use variables calculated on 'try'? Maybe we could comment this part, same as the "testing part" you mentioned above?

Reply:
Yup it's not used, it could be commented out. Will change :+1: